### PR TITLE
[0.16.2] Add release notes for lighthouse PR #1439

### PR DESCRIPTION
**Pull request by tpantelis**
Release notes for https://github.com/submariner-io/lighthouse/pull/1439

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ weight = 40
 
 ## v0.16.2
 
-* Fixed known issue in v0.16.1 where Globalnet did not work properly if deployed using the Helm charts.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md:10]: Please check why these two entries are removed.

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -7,8 +7,7 @@ weight = 40
...
-* Fixed known issue in v0.16.1 where `subctl verify` with Globalnet installed may fail with a panic.
+* Fixed an issue in Service Discovery which caused an erroneous ServiceExport Conflict status condition to be reported.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md:10]: This should be part of 0.16.3 isnt it?

**Review by sridhargaddam — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 2 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md:10]: yes

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md:10]: These were never merged to the main branch but we need to start a new section anyway so we can leave it.

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**